Makefile

```diff
@@ -129,6 +129,27 @@ gofail-disable: install-gofail
 install-gofail:
 	cd tools/mod; go install go.etcd.io/gofail@${GOFAIL_VERSION}
 
+# Reproduce historical issues
+
+.PHONY: reproduce-issue14685
+reproduce-issue14685: ./bin/etcd-v3.5.5-failpoints
+	cp ./bin/etcd-v3.5.5-failpoints ./bin/etcd
+	GO_TEST_FLAGS='-v --run=TestLinearizability/Issue14685 --count 100 --failfast' make test-linearizability
+
+./bin/etcd-v3.5.5-failpoints:
+	rm -rf /tmp/etcd-release-v3.5.5/
+	mkdir -p /tmp/etcd-release-v3.5.5/
+	cd /tmp/etcd-release-v3.5.5/; \
+	  git clone https://github.com/etcd-io/etcd.git .; \
+	  git checkout v3.5.5; \
```

Comment:
The comment https://github.com/etcd-io/etcd/pull/14912#discussion_r1044422353 applies to this PR as well.

Reply:
Marking this PR as draft until we resolve https://github.com/etcd-io/etcd/pull/14912#discussion_r1044422353

Reply:
Removed the script. PTAL.